Sources/Clappr/Classes/Base/Options.swift

```diff
@@ -17,6 +17,10 @@ public let kMediaControlAlwaysVisible = "mediaControlAlwaysVisible"
 /// List of MediaControl Plugins
 public let kMediaControlPlugins = "mediaControlPlugins"
```

Comment:
Hey guys, 
I know that this wasn't added in this Pull Request. 😅  But we need to understand that the concept of "Media Control Plugins" don't exist.
We have Core Plugins, not something apart for for media control.

If you need to add a MediaControl plugin and this is not possible for now without this option `kMediaControlPlugins`, let's think about rebasing Media Control branches on the Loader Pull request so we can use the method of `Player.register(plugins: plugins)`. Otherwise we are writing new code without following Clappr's architecture.

Reply:
As accorded we'll treat this issue at another card.